post_process_epics.sh

```diff
@@ -21,55 +26,97 @@ fi
 # jenkins core has around 600
 SEARCH_LIMIT=1000
 
+# TODO once search allows to check if there's a parent then skip those that already have a parent
```

Comment:
I couldn't see a way to get the parent from the gh search result so that once its been linked we skip it